packages/fs-detectors/src/workspaces/get-workspace-package-paths.ts

```diff
@@ -103,6 +106,17 @@ async function getPackageJsonWorkspacePackagePaths({
   return getPackagePaths(packages, fs);
 }
 
+async function getNxWorkspacePackagePaths({
+  fs,
+}: GetPackagePathOptions): Promise<string[]> {
+  const nxWorkspaceJsonAsBuffer = await fs.readFile('workspace.json');
+
+  const { projects } = JSON.parse(nxWorkspaceJsonAsBuffer.toString());
+
+  const packages: string[] = Object.values(projects) ?? [];
```

Comment:
In what case can `Object.values(projects)` be falsy? Seems like the `??` is not used.

Reply:
Just tested and that is the case that `??` is not used. Updated